File: JS/qlparser/src/main/java/nl/uva/js/qlparser/models/qls/Stylesheet.java
package nl.uva.js.qlparser.models.qls;

import lombok.Builder;
import lombok.Data;
import nl.uva.js.qlparser.models.qls.elements.Page;
import nl.uva.js.qlparser.models.qls.style.DefaultStyle;

import java.util.LinkedList;

@Data
@Builder
public class Stylesheet {
    private String name;
    private LinkedList<DefaultStyle> defaultStyles;
    private LinkedList<Page> pages;

    public void checkType() {
//
    }
}

File: niro/src/main/scala/nl/uva/se/sc/niro/qls/model/gui/Stylesheet.scala
package nl.uva.se.sc.niro.qls.model.gui

import nl.uva.se.sc.niro.ql.model.ast.AnswerType

/**
  * Model used by the frontend
  */
case class Stylesheet(name: String, pages: Seq[Page], defaultStyles: Map[AnswerType, Styling])
